docs/src/getstarted/hello-world.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Hello World Quickstart Guide"
+description: "This quickstart guide will demonstrate how to setup, build, and deploy your first Solana program in your browser with Solana Playground."
```

Comment:
Maybe mention `hello world` for SEO?

Reply:
I don't think it is needed for SEO purposes since it is already in the title, and mentions several times in the pages content